src/http_api_tool/verifier.py

```diff
@@ -34,10 +34,13 @@ def log(self, message: str, emoji: str = "") -> None:
             message = f"{message} {emoji}"
         print(message)
 
-    def debug_log(self, message: str) -> None:
+    def debug_log(self, message: str, sensitive_data: bool = False) -> None:
         """Log a debug message if debug mode is enabled."""
         if self.debug:
-            print(f"🐞 {message}")
+            if sensitive_data:
+                print(f"🐞 [REDACTED] {message}")
```

Comment:
if `sensitive_data` is true, then it still writes the message, just with `[REDACTED]` pre-pended? Doesn't that defeat the purpose?